server/rpc.go

```diff
@@ -285,10 +285,10 @@ func GetTranscoder(context context.Context, orch Orchestrator, req *TranscoderRe
 	job, err := orch.GetJob(req.JobId)
 	if err != nil {
 		glog.Error("Unable to get job ", err)
-		return nil, err
+		return nil, fmt.Errorf("Unable to get job (%s)", err.Error())
 	}
-	if !verifyTranscoderReq(orch, req, job) {
-		return nil, fmt.Errorf("Invalid transcoder request")
+	if ok, err := verifyTranscoderReq(orch, req, job); !ok || err != nil {
+		return nil, fmt.Errorf("Invalid transcoder request (%v)", err)
```

Comment:
Normally I would just `return nil, err` here for brevity since we know that the error is wrapped within `verifyTranscoderReq`, but that's up to you. Concatenating the errors could be somewhat useful to build a pseudo-backtrace if we ever need to call this function in several places.